diff --git a/lib/rules/forward-ref-uses-ref.js b/lib/rules/forward-ref-uses-ref.js
index 3a0b7de4c951e7ea92a9de7bac924ba95d643342..27f88d5782c9918f503be78b5b71add96941d8ff 100644
--- a/lib/rules/forward-ref-uses-ref.js
+++ b/lib/rules/forward-ref-uses-ref.js
@@ -57,7 +57,7 @@ module.exports = {
   },
 
   create(context) {
-    const sourceCode = context.getSourceCode();
+    const sourceCode = (context.sourceCode || context.getSourceCode());
 
     return {
       'FunctionExpression, ArrowFunctionExpression'(node) {
diff --git a/lib/rules/jsx-filename-extension.js b/lib/rules/jsx-filename-extension.js
index 4a2cfeede66b68255ca48bc38c0eb2f1537a785a..037a9c2f7f2f6f1b2fa789930d9a9a4d151b0c37 100644
--- a/lib/rules/jsx-filename-extension.js
+++ b/lib/rules/jsx-filename-extension.js
@@ -61,7 +61,7 @@ module.exports = {
   },
 
   create(context) {
-    const filename = context.getFilename();
+    const filename = (context.filename || context.getFilename());
 
     let jsxNode;
 
diff --git a/lib/util/eslint.js b/lib/util/eslint.js
index 79a0537f3b4f52b04b29d46e89ce00f6da8fe579..08713542539e064370a8c85ef670037a264fdb46 100644
--- a/lib/util/eslint.js
+++ b/lib/util/eslint.js
@@ -15,7 +15,7 @@ function getScope(context, node) {
     return sourceCode.getScope(node);
   }
 
-  return context.getScope();
+  return (context.sourceCode || {}).getScope ? context.sourceCode.getScope(node) : context.getScope();
 }
 
 function markVariableAsUsed(name, node, context) {
diff --git a/lib/util/version.js b/lib/util/version.js
index c9dc683b7d149c5a3a941bdcbd32f657facdfbf6..b2cb0ff93febcb34b3b5a5d6cfd4b35c94e4a643 100644
--- a/lib/util/version.js
+++ b/lib/util/version.js
@@ -28,7 +28,7 @@ function resetDetectedVersion() {
 
 function resolveBasedir(contextOrFilename) {
   if (contextOrFilename) {
-    const filename = typeof contextOrFilename === 'string' ? contextOrFilename : contextOrFilename.getFilename();
+    const filename = typeof contextOrFilename === 'string' ? contextOrFilename : (contextOrFilename.filename || contextOrFilename.getFilename());
     const dirname = path.dirname(filename);
     try {
       if (fs.statSync(filename).isFile()) {
