diff --git a/node_modules/check-links/lib/check-link.js b/node_modules/check-links/lib/check-link.js
index ccef218..9063e0c 100644
--- a/node_modules/check-links/lib/check-link.js
+++ b/node_modules/check-links/lib/check-link.js
@@ -85,13 +85,13 @@ export function checkLink(url, opts = {}) {
 
   return fetchHEAD()
     .catch((/** @type {HTTPError} */ err) => {
-      console.warn('HEAD error', err)
+      // console.warn('HEAD error', err)
       // TODO: if HEAD results in a `got.HTTPError`, are there status codes where
       // we can bypass the GET request?
       return fetchGET()
     })
     .catch((/** @type {Response} */ err) => {
-      console.warn('GET error', err)
+      // console.warn('GET error', err)
       return /** @type {const} */ ({
         status: 'dead',
         statusCode: err.statusCode
