diff --git a/node_modules/memfs/lib/index.d.ts b/node_modules/memfs/lib/index.d.ts
index 7eaabf6..8335054 100644
--- a/node_modules/memfs/lib/index.d.ts
+++ b/node_modules/memfs/lib/index.d.ts
@@ -1,6 +1,6 @@
 import Stats from './Stats';
 import Dirent from './Dirent';
-import { Volume as _Volume, StatWatcher, FSWatcher, IReadStream, IWriteStream, DirectoryJSON } from './volume';
+import { Volume as _Volume, StatWatcher, FSWatcher, IReadStream, IWriteStream, DirectoryJSON, IRealpathOptions, IMkdirOptions, IReaddirOptions, IStatOptions,TData, TMode } from './volume';
 import { IPromisesAPI } from './promises';
 import { constants } from './constants';
 export { DirectoryJSON };
@@ -19,3 +19,17 @@ export interface IFs extends _Volume {
 }
 export declare function createFsFromVolume(vol: _Volume): IFs;
 export declare const fs: IFs;
+
+
+export {
+  IRealpathOptions,
+  IMkdirOptions,
+  IReaddirOptions,
+  IStatOptions,
+  TData,
+  TMode,
+  _Volume
+};
+export * from './Dirent';
+export * from './Stats';
+export * from './encoding';
\ No newline at end of file
